.flake8.cython

```diff
@@ -0,0 +1,3 @@
+[flake8]
+filename = *.pyx,*.pxd,*.pxi
```

Comment:
Not sure if it's a typo or it's intentional, but we used to validate only `pyx` files for these errors, and the CI is failing because there are `pxd` and `pxi` files that got errors.

Reply:
Yes this is a typo. I separated it out to another `.flake8`, pushed the fix.